install/quick-start/.helpers.sh

```diff
@@ -193,9 +193,11 @@ install_control_plane() {
 
 # Install OpenChoreo Data Plane
 install_data_plane() {
+    local enable_observability="${1:-false}"
```

Comment:
Passing a Boolean argument to this helper function isn't consistent with others IMO.
Instead, shall we use the ENABLE_OBSERVABILITY flag itself? like other functions does.. (ex: https://github.com/openchoreo/openchoreo/pull/892/files#diff-103cddf3bc93e1c47f23b9b3ea102463fb59768db9157cb81f7268b49487dc65R294)